.github/workflows/linux.yml

```diff
@@ -16,476 +16,490 @@ concurrency:
+        test:
+          - name: 'VLM API'
+            cmd: 'tests/python_tests/test_vlm_api.py'
+          - name: 'Whisper Generate API'
+            cmd: 'tests/python_tests/test_whisper_generate_api.py -k "${PRECOMMIT_FILTER}"'
     timeout-minutes: 60
     defaults:
       run:
         shell: bash
-    runs-on: ubuntu-20.04
-    env:
-      CMAKE_BUILD_PARALLEL_LEVEL: null
-      OV_INSTALL_DIR: ${{ github.workspace }}/ov
-      CCACHE_DIR: ${{ github.workspace }}/ccache
-      CCACHE_MAXSIZE: 500Mi
-      CMAKE_CXX_COMPILER_LAUNCHER: ccache
-      CMAKE_C_COMPILER_LAUNCHER: ccache
+    runs-on: aks-linux-4-cores-16gb
+    container:
+      image: openvinogithubactions.azurecr.io/ov_test/ubuntu_22_04_x64:${{ needs.openvino_download.outputs.docker_tag }}
+      volumes: 
+        - /mount:/mount
+        - ${{ github.workspace }}:${{ github.workspace }}
 
+    env:
+      INSTALL_DIR: ${{ github.workspace }}/install
+      SRC_DIR: ${{ github.workspace }}/src
+      BUILD_DIR: ${{ github.workspace }}/build
+      PRECOMMIT_FILTER: 'not (test_language or timestamps)'
+        
     steps:
       - name: Clone openvino.genai
-        uses: actions/checkout@v4
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
+          path: ${{ env.SRC_DIR }}
           submodules: recursive
-
-      - name: Setup Python ${{ env.PYTHON_VERSION }}
-        uses: actions/setup-python@v5
-        with:
-          python-version: ${{ env.PYTHON_VERSION }}
-          cache: 'pip'
-
-      - name: Download OpenVINO package
-        uses: actions/download-artifact@v4
+          
+      - name: Download Build Artifacts
+        uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4.1.8
         with:
-          name: openvino_package
-          path: ${{ env.OV_INSTALL_DIR }}
-
-      - name: Extract OpenVINO packages
-        run: |
-          pushd ${OV_INSTALL_DIR}
-            tar -xzf openvino_package.tar.gz -C ${OV_INSTALL_DIR} --strip-components=1
-          popd
-
-      - name: Set apt
-        run: |
-          echo 'Acquire::Retries "10";' | sudo tee -a /etc/apt/apt.conf.d/80-retries > /dev/null
-          echo 'APT::Get::Assume-Yes "true";' | sudo tee -a /etc/apt/apt.conf.d/81-assume-yes > /dev/null
-          echo 'APT::Get::Fix-Broken "true";' | sudo tee -a /etc/apt/apt.conf.d/82-fix-broken > /dev/null
-          echo 'APT::Get::no-install-recommends "true";' | sudo tee -a /etc/apt/apt.conf.d/83-no-recommends > /dev/null
-
-      - name: Install build dependencies
+          pattern: "{${{ needs.openvino_download.outputs.ov_artifact_name }},genai_wheels}"
+          path: ${{ env.INSTALL_DIR }}
+          merge-multiple: true
+          
+      - name: Install GenAI Wheels
+        run: python -m pip install ${{ env.INSTALL_DIR }}/wheels/openvino_genai*.whl openvino-tokenizers[transformers] ${{ needs.openvino_download.outputs.ov_wheel_source }} --force-reinstall
```

Comment:
We need to install `openvino-tokenizers[transformers]` from local file system similarly to `openvino_genai*.whl`